.github/workflows/validate-nightly-binaries.yml

```diff
@@ -0,0 +1,31 @@
+name: Validate nightly binaries (all OS)
+
+on:
+  schedule:
+    # At 2:30 pm UTC (7:30 am PDT)
```

Comment:
Would be good to add a comment about rough estimation of when these would complete, i.e. how long do these test take to finish.

Reply:
Do you think it would be sufficient to add the estimates to the PR description (since the workflow contains the link to the PR). I just don’t want to trigger the jobs for such a minor change.

Reply:
Updated PR description with approximate timing: 1:40-2 hours and the link to an example run.